assets/src/dashboard/app/api/wpAdapter.js

```diff
@@ -28,11 +33,21 @@ const post = (path, options = {}) =>
     method: 'POST',
   });
 
+// `apiFetch` by default turns `DELETE` requests into `POST` requests
+// with `X-HTTP-Method-Override: DELETE` headers.
+// However, some Web Application Firewall (WAF) solutions prevent this.
+// `?_method=DELETE` is an alternative solution to override the request method.
+// See https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_method-or-x-http-method-override-header
 const deleteRequest = (path, options = {}) =>
   apiFetch({
-    path,
+    path: queryString.stringifyUrl({
```

Comment:
Can we use [addQueryArgs](https://github.com/google/web-stories-wp/blob/f5a891e6db755d61b39dc53300ddf8b1e4aea973/assets/src/edit-story/utils/addQueryArgs.js#L30-L54) here?

Reply:
`addQueryArgs` util is only available in the editor. The dashboard has been using `queryString.stringifyUrl` throughout the code base.

Might be worth unifying that in a future PR